src/main/java/com/grim3212/mc/pack/industry/config/IndustryConfig.java

```diff
@@ -78,6 +81,7 @@ public void syncConfig() {
 
 		registerWorkbenchList(workbenchList, IndustryConfig.workbenchUpgradeList);
 
+		uraniumDamage = config.get(CONFIG_GENERAL_NAME, "Damage from Uranium Ore", 4).getInt();
```

Comment:
Also, just so you know each config option has a tooltip that I define in the lang file.